pylint/pyreverse/main.py

```diff
@@ -76,7 +77,7 @@
         "all-ancestors",
         dict(
             short="A",
-            default=None,
+            default="",
```

Comment:
@DudeNr33 Could you tell me if these default values make sense? Tests do pass but I'm not sure how extensive our `pyreverse` tests are..

Reply:
@DudeNr33 Gentle ping as this is blocking some of my other PRs for `argparse`. Never mind if you just didn't have the time but my recent work has probably flooded most of our inboxes and this might have gone unnoticed 😄

Reply:
Just saw it like a minute before you pinged me! 😁 I probably won't have time to look at it thoroughly today, but will try to tomorrow!

Reply:
``"all-ancestors"`` and ``"all-associated"`` are used as simple flags. 
Looking at ``OptionsProviderMixin.load_defaults()`` we should either leave this at ``None`` or remove the ``default`` key: ``load_defaults`` calls ``set_option`` with ``optdict.get("default")`` for the ``value`` parameter. 

We would then end up with ``""`` as value for this option, but the ``pyreverse`` explicitly checks for ``is None`` (in ``DiaDefGenerator._set_option``).